services/galaxytoolshed/galaxytoolshed-activity.service.js

```diff
@@ -0,0 +1,41 @@
+import { age } from '../color-formatters.js'
+import { formatDate } from '../text-formatters.js'
+import BaseGalaxyToolshedService from './galaxytoolshed-base.js'
+
+export default class GalaxyToolshedCreatedDate extends BaseGalaxyToolshedService {
+  static category = 'activity'
+  static route = {
+    base: 'galaxytoolshed/created-date',
+    pattern: ':repository/:owner',
+  }
+
+  static examples = [
+    {
+      title: 'Galaxy Toolshed (created time)',
+      namedParams: {
+        repository: 'sra_tools',
+        owner: 'iuc',
+      },
+      staticPreview: this.render({
+        date: this.render({ date: '2022-01-01' }),
+      }),
+    },
+  ]
+
+  static defaultBadgeData = {
+    label: 'created date',
+  }
+
+  static render({ date }) {
+    return { message: formatDate(date), color: age(date) }
```

Comment:
I think for the color we should just `blue` since this is an informational point. The `age` color helper applies a color scale that's useful for updates because it can reflect activity (i.e. this package is regularly updated/maintained vs. it hasn't been updated in 10 years).

In this case though, we're just displaying the static date when the package was first created so using this color scale would essentially penalize older/more established packages with a red color.

Let me know if I've misunderstood, but if the above is correct then I'd suggest we do this instead, along with an extra line up above in defaultBadgeData

```suggestion
    return { message: formatDate(date) }
```

Reply:
Sorry for the delay,
It's true, the modifications are done in 353338867